### ISSUE TITLE
Cannot set txpower higher than 20dB

### ISSUE THREAD

**Issue by hifihedgehog** [aircrack-ng/rtl8814au#2]
See above. In Debian Buster, I cannot set the TX power above 20 db using the command sudo iw wlx9cc9eb22d14a set txpower fixed 3000. It stays fixed at Tx-Power=20 dBm.

**Comment by kimocoder** [comment]
That's right. This capabilities has not been added yet, it's not going to be 1st priority

**Comment by raplin** [comment]
Just submitted a PR that reinstates the code from 88XXAU for power control, works (4000=100%)
